### PR TITLE
chore: generator 7.7.3 update

### PR DESCRIPTION
**Pull request by brandonocasey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "index.html",
     "scripts/",
     "src/",
-    "test/"
+    "test/",
+    "bin"
```
Comment (brandonocasey) [inline package.json:25]: master cannot be released right now as `bin` does not get published.

File: scripts/rollup.config.js [hunk 2]
```diff
@@ -7,45 +7,32 @@ const config = generate(options);
...
-    plugins: config.builds.browserMin.plugins
-  });
-
-  config.builds.unpadMin = config.makeBuild('browser', {
+    }]
+  }),
+  unpad: config.makeBuild('browser', {
     input: 'src/unpad.js',
     output: [{
+      name: 'pkcs7.unpad',
```
Comment (brandonocasey) [inline scripts/rollup.config.js:26]: we minify all files named `.min.js` now. so we can make the build much faster by using two outputs.

File: test/pkcs7.test.js [hunk 3]
```diff
@@ -49,9 +49,11 @@ QUnit.test('works on buffers greater than sixteen bytes', function(assert) {
 
   assert.expect(2);
 
-  assert.equal(pkcs7.pad(buffer).length - buffer.length,
+  assert.equal(
+    pkcs7.pad(buffer).length - buffer.length,
```
Comment (brandonocasey) [inline test/pkcs7.test.js:53]: liner fixes.

File: package.json [hunk 4]
```diff
@@ -30,53 +31,38 @@
     "videojs-plugin"
   ],
   "dependencies": {
-    "global": "^4.3.2"
+    "@babel/runtime": "^7.5.5"
   },
   "devDependencies": {
-    "conventional-changelog-cli": "^2.0.1",
```
Comment (gkatsev) [inline package.json:36]: all of these are part of the generator-helpers?

File: package.json (hunk 1 above)
Comment (gkatsev) [reply package.json:25]: nobody really uses the bin :P

**Review by gkatsev — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (brandonocasey) [reply package.json:25]: yeah it just fails to install

File: package.json (hunk 4 above)
Comment (brandonocasey) [reply package.json:36]: yeah

**Pull request merged**